databend-jdbc/src/main/java/com/databend/jdbc/DatabendConnection.java

```diff
@@ -348,6 +348,11 @@ public int getHoldability() throws SQLException {
         return 0;
     }
 
+    // TODO(zhihanz): allow config through jdbc params
+    public int getFailOver() {
```

Comment:
getFailOver -> getMaxFailoverRetries ? might makes it more clear about it's a maximium limit instead of a policy